composer.json

```diff
@@ -38,7 +38,7 @@
         "appwrite/php-clamav": "1.1.*",
         "appwrite/php-runtimes": "0.6.*",
 
-        "utopia-php/framework": "0.19.*",
+        "utopia-php/framework": "0.19.1",
```

Comment:
This should be fixed now, you can use `0.19.*` again 👍🏻

Reply:
Perfect 👌